apps/src/lab2/projects/ProjectManager.ts

```diff
@@ -113,6 +92,23 @@ export default class ProjectManager {
     return {sources, channel};
   }
 
+  async restore(versionId: string): Promise<ProjectSources | undefined> {
```

Comment:
nit: mind adding a doc explaining what this function does? mostly so it's clear that it's performing a project overwrite (not just retrieving)

Also would `restoreSources` or `revertSources` be a bit clearer?

Reply:
Will update!